src/main/java/com/julienvey/trello/Trello.java

```diff
@@ -79,6 +79,10 @@
 
     TList getList(String listId, Argument... args);
 
+    /* CheckLists */
+
+    CheckList getCheckList(String checkListId, Argument... args);
```

Comment:
I'm not sure I agree. It's a top-level operation in the API. Not specific to a card.